tiles/src/main/java/com/protomaps/basemap/layers/Pois.java

```diff
@@ -17,21 +17,68 @@ public String name() {
 
   @Override
   public void processFeature(SourceFeature sf, FeatureCollector features) {
-    if (sf.isPoint() && (sf.hasTag("amenity") ||
-      sf.hasTag("shop") ||
-      sf.hasTag("tourism") ||
-      sf.hasTag("railway", "station"))) {
-      var feature = features.point(this.name())
-        .setId(FeatureId.create(sf))
-        .setAttr("amenity", sf.getString("amenity"))
-        .setAttr("shop", sf.getString("shop"))
-        .setAttr("railway", sf.getString("railway"))
-        .setAttr("cuisine", sf.getString("cuisine"))
-        .setAttr("religion", sf.getString("religion"))
-        .setAttr("tourism", sf.getString("tourism"))
-        .setZoomRange(13, 15);
-
-      OsmNames.setOsmNames(feature, sf, 0);
+    if( !sf.canBeLine() && (sf.isPoint() || sf.canBePolygon())
```

Comment:
We want to also harvest polygons labels from OSM ways and relations, not just loose POIs from OSM nodes.